### ISSUE TITLE
use standard fdroid repo dir layout

### ISSUE THREAD

**Issue by eighthave** [NoMore201/playmaker#16]
This is great work!  I have one small request that'll make this integrate better into the F-Droid ecosystem: please use the standard directory layout.  It seems right now this puts the repo straight into _fdroid/_.  Instead it should look like:
```
-fdroid
       |-config.py
       \-repo
             |-index.xml
             |-my.package_123.apk
```

**Comment by NoMore201** [comment]
I'm quite sure I'm using the standard fdroid layout. During initialization, the following commands are executed: 

```
fdroid init -v
fdroid update --create-key -v
```
The last call is redundant for first time initialization, but it is useful in case the repo was already initialized but it's missing some required files.

This is the result of initialization:

```
pm-dir
        |-config.py
        |-archive/
        |-fdroid-icon.png
        |-keystore.jks
        |-metadata/
        |-srclibs/
        |-tmp/
        \-repo
            |-index-xml
            |-123.apk
            [...]
```

**Comment by eighthave** [comment]
Yup, that looks standard.  Perhaps you are using something different
than `fdroid server update` to publish the repo to the webserver?

Also, you should check out the gitlab/github repo hosting.  You could
have Playmaker pushing repos to github for free hosting.  See
examples/config.py.

**Comment by Yannik** [comment]
@eighthave Which issue are you actually reporting here? 
That the repo is being served at `domain.org/fdroid` instead of `domain.org/repo`? Apart from that, the standard fdroid repo setup is being used.

**Comment by eighthave** [comment]
The full repo should have a basedir of _fdroid/_, e.g. `domain.org/fdroid`,
then there should be a "repo" and "archive" sections there:

* domain.org/fdroid/repo/index.xml
* domain.org/fdroid/archive/index.xml

**Comment by eighthave** [comment]
Then `domain.org/fdroid/index.html` can optionally be used as a webpage to describe the repo.

**Comment by NoMore201** [comment]
Since the current setup works, I guess that the fdroid app will check both these paths:

- `<playmaker_host>/fdroid/index.xml`
- `<playmaker_host>/fdroid/repo/index.xml`

right? If so, in the next version I'll change the base directory. The `index.html` thing i think could be done manually by users

**Comment by eighthave** [comment]
Only fdroid/repo/index.xml is supported as a standard location.

**Comment by eighthave** [comment]
And yeah, the index.html would be done manually by the users.  It would
be nice to have an auto-generated version, but someone would have to
implement that :-)
